posthog/migrations/0230_cohort_filters.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.2.12 on 2022-04-28 13:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0229_add_filters_hash_to_dashboard_table"),
+    ]
+
+    operations = [
+        migrations.AddField(model_name="cohort", name="filters", field=models.JSONField(null=True),),
```

Comment:
One look here - I think a migration like this should be fine 👀

Reply:
Should we make this blank=True too?

Reply:
.. Not sure. Don't think we'd ever want to send in `''` to this field?

Reply:
I don't think it makes a difference? I'm just asking because we've set up a standard where adding model fields should always be null=True and blank=True

Reply:
I see, sure.

Although, let me convert this into a single migration instead 😅